x-pack/docs/en/security/securing-communications/node-certificates.asciidoc

```diff
@@ -75,12 +75,27 @@ signing requests (CSR) for the nodes in your cluster. For more information, see
 <<certutil>>.
 --
 
-. Copy the node certificate to the appropriate locations.
+. Optional: Generate additional certificates specifically for encrypting HTTP
+client communications. 
 +
 --
-Copy the applicable `.p12` file into a directory within the {es} configuration
-directory on each node. For example, `/home/es/config/certs`. There is no need
-to copy the CA file to this directory.
+For example, use the `elasticsearch-certutil http` command:
+
+[source,shell]
+----------------------------------------------------------
+bin/elasticsearch-certutil http
+----------------------------------------------------------
+
+This command guides you through the process of generating the appropriate
+certificates for use in {es} and {kib}. If you created a CA for your cluster,
+you can re-use it by supplying its location when prompted.
+--
+
+. Copy the node certificates to the appropriate locations.
++
+--
+Copy the applicable `.p12` files into the {es} configuration directory on each
+node. There is no need to copy the CA file to this directory.
```

Comment:
> There is no need to copy the CA file to this directory.

Not sure what we originally meant with this, I guess we assumed that the CA certificate is part of the PKCS#12 that we would create and then users wouldn't need to copy the CA certficate by itself in addition. In any case, I feel this makes assumptions on the setup and is not very helpful so why not take the opportunity to remove this sentence